types/lib/query-interface.d.ts

```diff
@@ -120,6 +120,15 @@ export interface QueryInterfaceDropAllTablesOptions extends QueryInterfaceOption
   skip?: string[];
 }
 
+export interface TablenameWithSchema {
```

Comment:
Should probably be `TableName`.

Reply:
Do you mean it should be TableName or TableNameWithSchema?

Reply:
He probably means `TableNameWithSchema`